packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -327,6 +400,25 @@ function Preview({ isInspecting, setIsInspecting, zoomLevel, onZoomChanged }: Pr
     device: device!,
   });
 
+  const TouchPointMarker: React.FC<TouchPoint> = ({ x, y }) => {
+    const styles: React.CSSProperties = {
```

Comment:
Please move it outside of Preview so it doesn't get redefined every render. You also [don't need](https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/) `React.FC` here. If you move `styles` inline, you don't need to type it, we don't get much from that variable anyway.